pom.xml

```diff
@@ -3,10 +3,11 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.33</version>
+    <version>3.50</version>
     <relativePath />
   </parent>
   <properties>
+    <java.level>7</java.level>
```

Comment:
I would recommendgoing after Java 8 and 2.60+ , but it is up to maintainers. We are deprecating the Java 7 support in Plugin POM